methods/messages/redactMessageText.md

```diff
@@ -0,0 +1,48 @@
+{% method %}
+## Redact Message Text information
+This endpoint can be used to redact the text of a previously sent message. You will need to send the catapult user-id and the message id (of the message for which you wish to redact the content) in the request URL.
```

Comment:
```You will need to send the catapult user-id and the message id (of the message for which you wish to redact the content) in the request URL.```
We don't get this wordy in the docs for other endpoints. The fact that a `userId` and `messageId` are required is demonstrated by the sample URL below. We also probably should not use the term "catapult user-id" anyway. All customer facing material uses the term "Application Platform" rather than "Catapult".

I would recommend removing this whole sentence.